res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
It is essential to use "GPL 2 or later" if we want to be compatible with code licensed under ASL2, which includes *lots* of code. Certainly fred needs to be GPL-2-or-later, otherwise it is technically illegal for us to use ASL2 code such as the Apache Commons Compress library (which we do use in fred). I don't think the installer uses such code, however.

It seems very unlikely that any contributor would sue FPI to get us to release the Apple signing key. Even if they did, the publicity would do wonders for us. It's much more likely that Apple prohibits GPL3 code from running at all, but again, there isn't much to lose IMHO.